app/src/main/java/org/onionshare/android/files/FileManager.kt

```diff
@@ -159,18 +143,16 @@ class FileManager @Inject constructor(
                 }
             }
         } catch (e: CancellationException) {
+            LOG.info("Got cancelled, deleting zip file...")
             zipFile.delete()
             throw e
         }
         currentCoroutineContext().ensureActive()
```

Comment:
I'm wondering what happens here if the task is cancelled while we're zipping the last file. I'm guessing we won't catch a `CancellationException` above because we called `ensureActive()` before zipping the last file, thus before the task was cancelled. So would we throw a `CancellationException` at this line, without having called `zipFile.delete()` above or `zipFile.deleteOnExit()` below?

Maybe this is #10 and out of scope here?

Reply:
Yeah, I'd leave this for #10.